### PR TITLE
Add CMI Comprehensive  metrics support docs

### PR DESCRIPTION
**Pull request by ravikiran-sulikeri**
[Trello](https://trello.com/c/1BIA7N0q/2422-document-az-secondaries-changes-in-cmi)

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-437.surge.sh

Updated pages:
- [Customer Metrics Integration (CMI)](https://neo4j-docs-aura-437.surge.sh/aura/platform/metrics-integration)

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 1]
```diff
@@ -236,6 +236,58 @@ aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_u
...
+# HELP neo4j_aura_cpu_usage CPU usage (cores). CPU is used for planning and serving queries. If this metric is constantly spiking or at its limits, consider increasing the size of your instance.
+# TYPE neo4j_aura_cpu_usage gauge
+neo4j_aura_cpu_usage{aggregation="MAX",instance_id="a59d71ae",availability_zone="eu-west-1b",instance_mode="PRIMARY"} 0.047088 1724245310000
+# HELP neo4j_aura_cpu_usage CPU usage (cores). CPU is used for planning and serving queries. If this metric is constantly spiking or at its limits, consider increasing the size of your instance.
+# TYPE neo4j_aura_cpu_usage gauge
+neo4j_aura_cpu_usage{aggregation="MAX",instance_id="a59d71ae",availability_zone="eu-west-1c",instance_mode="PRIMARY"} 0.021874 1724245310000
+# HELP neo4j_aura_cpu_usage CPU usage (cores). CPU is used for planning and serving queries. If this metric is constantly spiking or at its limits, consider increasing the size of your instance.
+# TYPE neo4j_aura_cpu_usage gauge
+neo4j_aura_cpu_usage{aggregation="MAX",instance_id="a59d71ae",availability_zone="eu-west-1a",instance_mode="SECONDARY"} 0.043957 1724245304000
```
Comment (alexnb) [inline modules/ROOT/pages/platform/metrics-integration.adoc:262]: I would remove all the HELP and TYPE lines to focus on the contents.

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 2]
```diff
@@ -236,6 +236,58 @@ aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_u
...
+----
+
+.Additional Metric labels
+* `availability_zone` - User selected Cloud provider zone.
+* `instance_mode` - `PRIMARY` or `SECONDARY` based on user selected workload requirement. (Minimum 3 primaries per instance)
+
+.Usage
+Here is an example of gaining more insights into your Aura instance CPU usage for capacity planning:
```
Comment (alexnb) [inline modules/ROOT/pages/platform/metrics-integration.adoc:270]: Maybe mentioning which expression language it is (assume it is Datadog?)

**Review by alexnb — APPROVED** [review]
Nice! :rocket: See my comments.

File: modules/ROOT/pages/platform/metrics-integration.adoc (hunk 2 above)
Comment (ravikiran-sulikeri) [reply modules/ROOT/pages/platform/metrics-integration.adoc:270]: It's `promtail`. Will mention that.

**Comment by ravikiran-sulikeri** [comment]
> And we could also consider using light mode screenshots to match the other ones in that doc.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-437.surge.sh

Updated pages:
- [Customer Metrics Integration (CMI)](https://neo4j-docs-aura-437.surge.sh/aura/platform/metrics-integration)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-437.surge.sh

Updated pages:
- [Customer Metrics Integration (CMI)](https://neo4j-docs-aura-437.surge.sh/aura/platform/metrics-integration)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-437.surge.sh

Updated pages:
- [Customer Metrics Integration (CMI)](https://neo4j-docs-aura-437.surge.sh/aura/platform/metrics-integration)

**Review by alexnb — APPROVED** [review]

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 3]
```diff
@@ -236,6 +236,50 @@ aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_u
 
 * Reference: link:https://neo4j.com/labs/aura-cli/1.0/cheatsheet/[Aura CLI cheetsheet^]
 
+[aura-cmi-metrics-granularity]
+== Metrics Granularity
+
+Metrics returned by integration endpoint have the basic granularity reflective of the labels exposed, namely `aggregation`, `instance_id`, `database`.
+
+An Aura instance is typically underpinned by more than one nodes to achieve availability and workload scalability which typically get deployed into more than one, user selected, Cloud Provider availability zones in Primary or Secondary modes based on the workload requirements.
```
Comment (eve-bright) [inline modules/ROOT/pages/platform/metrics-integration.adoc:244]:
```suggestion
An Aura instance typically runs on multiple nodes to achieve availability and workload scalability. 
These nodes are deployed across different Cloud Provider availability zones in the user selected region. 
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-437.surge.sh

Updated pages:
- [Customer Metrics Integration (CMI)](https://neo4j-docs-aura-437.surge.sh/aura/platform/metrics-integration)

File: modules/ROOT/pages/platform/metrics-integration.adoc (hunk 3 above)
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/metrics-integration.adoc]:
```suggestion
An Aura instance is typically underpinned by more than one server to achieve availability and workload scalability.
These typically get deployed into more than one, user-selected, Cloud Provider availability zones in Primary or Secondary modes based on the workload requirements.
```

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 4]
```diff
@@ -236,6 +236,50 @@ aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_u
...
+
+Metrics Integration supports a more granular view of the Aura instance metrics with additional data points for availability zone & instance mode combinations. This view can be enabled on demand.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/metrics-integration.adoc:247]:
```suggestion
Metrics Integration supports a more granular view of the Aura instance metrics with additional data points for availability zone & instance mode combinations. 
This view can be enabled on demand.
```

File: modules/ROOT/pages/platform/metrics-integration.adoc (hunk 3 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/platform/metrics-integration.adoc]: We don't use _node_ to refer to cluster members.

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 5]
```diff
@@ -236,6 +236,50 @@ aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_u
...
+
+[NOTE]
+====
+Please contact link:https://support.neo4j.com/[Customer Support] to enable more granular metrics of instances for your tenant.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/metrics-integration.adoc:251]:
```suggestion
Contact link:https://support.neo4j.com/[Customer Support] to enable more granular metrics of instances for your tenant.
```

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 6]
```diff
@@ -236,6 +236,45 @@ aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_u
 
 * Reference: link:https://neo4j.com/labs/aura-cli/1.0/cheatsheet/[Aura CLI cheetsheet^]
 
+[aura-cmi-metrics-granularity]
+== Metrics Granularity
+
+Metrics returned by integration endpoint have the basic granularity reflective of the labels exposed, namely `aggregation`, `instance_id`, `database`.
+
+An Aura instance typically runs on multiple nodes to achieve availability and workload scalability.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/metrics-integration.adoc:244]:
```suggestion
An Aura instance typically runs on multiple servers to achieve availability and workload scalability. 
```

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 7]
```diff
@@ -236,6 +236,45 @@ aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_u
 
 * Reference: link:https://neo4j.com/labs/aura-cli/1.0/cheatsheet/[Aura CLI cheetsheet^]
 
+[aura-cmi-metrics-granularity]
+== Metrics Granularity
+
+Metrics returned by integration endpoint have the basic granularity reflective of the labels exposed, namely `aggregation`, `instance_id`, `database`.
+
+An Aura instance typically runs on multiple nodes to achieve availability and workload scalability. 
+These nodes are deployed across different Cloud Provider availability zones in the user selected region.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/metrics-integration.adoc:245]:
```suggestion
These servers are deployed across different Cloud Provider availability zones in the user-selected region. 
```

File: modules/ROOT/pages/platform/metrics-integration.adoc (hunk 6 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/platform/metrics-integration.adoc:244]: We don't use _node_ to refer to cluster members.

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 8]
```diff
@@ -236,6 +236,45 @@ aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_u
...
+neo4j_aura_cpu_usage{aggregation="MAX",instance_id="a59d71ae",availability_zone="eu-west-1b",instance_mode="PRIMARY"} 0.047088 1724245310000
+neo4j_aura_cpu_usage{aggregation="MAX",instance_id="a59d71ae",availability_zone="eu-west-1c",instance_mode="PRIMARY"} 0.021874 1724245310000
+----
+
+.Additional Metric labels
+* `availability_zone` - User selected Cloud provider zone.
+* `instance_mode` - `PRIMARY` based on user selected workload requirement of reads and writes. (Minimum 3 primaries per instance)
+
+.Usage
+Here is an example of gaining more insights into your Aura instance CPU usage for capacity planning:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/metrics-integration.adoc:267]:
```suggestion
The following is an example of gaining more insights into your Aura instance CPU usage for capacity planning:
```

**Review by AlexicaWright — COMMENTED** [review]
Some suggestion for readability.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-437.surge.sh

Updated pages:
- [Customer Metrics Integration (CMI)](https://neo4j-docs-aura-437.surge.sh/aura/platform/metrics-integration)

**Review by AlexicaWright — APPROVED** [review]
LGTM 👍 !Thank you @ravikiran-sulikeri

**Pull request merged**